scripts/common/k3s.sh

```diff
@@ -597,3 +598,14 @@ EOF
 
     source /etc/profile
 }
+
+KNOWN_VARS_FILE="$KURL_INSTALL_DIRECTORY/kurl.env"
```

Comment:
only worry is if we start storing multiple things here IMO (in case they stomp each other)
but it's fine for now

Reply:
yes, totally agree.  it will need to be atomic. load once in the beginning and save once at the end